### ISSUE TITLE
Should be add new berries ?

### ISSUE THREAD

**Issue by programgames** [PokeAPI/pokeapi#1126]
Can I create a Pull request to add new berries from gen 6 ?
https://bulbapedia.bulbagarden.net/wiki/Kee_Berry
https://bulbapedia.bulbagarden.net/wiki/Maranga_Berry
https://bulbapedia.bulbagarden.net/wiki/Hopo_Berry
https://bulbapedia.bulbagarden.net/wiki/Roseli_Berry

As i can see there is no information of firmness, smoothness ... on these new berries as berry mechanics have changed.

**Comment by Naramsim** [comment]
Yes! If you could submit a PR that would be appreciated! As for the non available data, is not there because Nintendo hasn't released it yet, or because they will never have such properties?

**Comment by FallenDeity** [comment]
completed with #1572
